src/GlobalSearch/GlobalSearchResultList.js

```diff
@@ -38,7 +39,8 @@ const GlobalSearchResultList = ({
     data.map(prisoner => (
       <tr key={`${prisoner.offenderNo}-${prisoner.uiId}`} className="row-gutters">
         <td className="row-gutters">
-          {prisoner.latestLocationId === agencyId ? (
+          {(viewInactivePrisoner && prisoner.currentlyInPrison === 'N') ||
```

Comment:
Do you think it's worth moving this tiny bit of duplication to a function? 
`
const shouldAddLink = (viewInactivePrisoner, caseLoads, prisoner) => viewInactivePrisoner && prisoner.currentlyInPrison === 'N') ||caseLoads.find(cl => cl === prisoner.latestLocationId)
`

Reply:
yes agreed!